application/models/models.py

```diff
@@ -189,3 +189,65 @@ def __init__(self, instrument_id=None, file_name=None, length=None, data=None):
         self.instrument_id = instrument_id
         self.file_name = file_name
         self.len = length
+
+
+class RegistryInstrumentModel(Base):
+    """This models the 'registry_instrument' table which holds the CIR instrument versions
+    selected by the Rops user for each EQ classifier for a given collection exercise.
+
+    * Keyed on the exercise_id, classifier_type and classifier_value, so for any given exercise_id, there can be
+      only one form_type classifier for a given classifier_value
+    * We currently only support "classifier_type": "form_type" (e.g. where "classifier_value" is "0001")
+    * The registry_instrument will only support a single classifier (as text)
+      and not multiple classifier combinations (as json)
+    * In RASRM the survey_id is a UUID, however in the context of the CIR object being returned, their survey_id is
+      what we know of as the three digit survey_ref e.g. "139". This table will hold OUR survey_id as a UUID and
+      not the three digit survey_ref returned by the CIR API.
+    """
+
+    __tablename__ = "registry_instrument"
+
+    # survey_id = Column(UUID, ForeignKey("survey.survey_id"))
+    # exercise_id = Column(UUID, ForeignKey("exercise.exercise_id"), primary_key=True)
+    # instrument_id = Column(UUID, ForeignKey("instrument.instrument_id"))
+    survey_id = Column(UUID, nullable=False)
+    exercise_id = Column(UUID, primary_key=True)
+    instrument_id = Column(UUID, nullable=False)
+    classifier_type = Column(String, primary_key=True)
+    classifier_value = Column(String, primary_key=True)
+    ci_version = Column(Integer, nullable=False)
+    guid = Column(UUID, nullable=False)
+    published_at = Column(TIMESTAMP, nullable=False)
+
+    def __init__(
+        self,
+        survey_id=None,
+        exercise_id=None,
+        instrument_id=None,
+        classifier_type=None,
+        classifier_value=None,
+        ci_version=None,
+        guid=None,
+        published_at=None,
```

Comment:
I'm thinking about introducing a DTO class to represent these as I'm handing them round quite a lot. A bit smelly.